src/plugins/kibana_react/public/code_editor/code_editor.tsx

```diff
@@ -357,6 +357,15 @@ export const CodeEditor: React.FC<Props> = ({
     ]
   );
 
+  useEffect(() => {
```

Comment:
Is it possible to migrate these changes to the shared-ux package in packages/shared-ux/code_editor/impl/code_editor.tsx. AFAIK we are hoping to de-duplicate the code from kibana react to the shared ux component (that was originally taken from this component).

Reply:
Heya @rshen91! It would great to consolidate and migrate all of these `CodeEditor`s to the new package, but I would like to do it in a future PR. I want to avoid having all my reviewers/testers having to re-test/re-review as this has to ship with 8.9.

Would it be beneficial if I copied the new code into the shared-ux `CodeEditor`  in this PR or should we wait until the consolidation?

Reply:
all good to do in a future PR!

Reply:
++ Thank you!